policy/lib/image.rego

```diff
@@ -23,6 +23,8 @@ parse(ref) := d if {
 		),
 	)
 
+	not contains(repo, "://")
```

Comment:
I'm assuming `oci://` is not a thing in this context, right? Or `docker://`.

Reply:
I haven't seen that being used outside of skopeo, so I'm not expecting it here

Reply:
Agree.